release-notes.md

```diff
@@ -399,6 +408,85 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v50.0
+
+This release has been tracked in [v50.0
+group](https://github.com/orgs/cloud-hypervisor/projects/6/views/6?filterQuery=release%3A%22Release+50%22)
+of our [roadmap project](https://github.com/orgs/cloud-hypervisor/projects/6/).
+
+### Configurable Nested Virtualization Option on x86_64
+
+The `nested=on|off` option has been added to `--cpu` to allow users
+to configure nested virtualization support in the guest on x86_64
+hosts (for both KVM and MSHV). The default value is `on` to maintain
+consistency with existing behavior. (#7408)
+
+### Compression Support for Qcow2
+
+Qcow2 support has been extended to handle compression clusters based on
```

Comment:
Would be worth double checking the "official" capitalisations of QCOW2, Zstd, etc,